patches/freebsd11_osymlink.patch

```diff
@@ -0,0 +1,95 @@
+--- ./lib/libc/sys/open.2.orig	2016-09-28 16:25:57.000000000 -0700
```

Comment:
To spin this differently... @mattklein123: now, that we have LDS and overall better xDS support, do you think that we still need the runtime configuration (and therfore `Watcher`)?